source/index.html.erb.md

```diff
@@ -3,12 +3,12 @@ title: "Tips for Getting Started with Web Accessibility"
 status: editors-draft
 ---
 
-Follow these introductory tips to help you get started with web accessibility. The tips are not all directly related to requirements but represent best practice that will help improve accessibility generally. The tips also provide links to further guidance, such as the WAI guidelines, tutorials, and educational resources.
+Get started with web accessibility, even if you only know little about it! These tips introduce some basic considerations, and link to further guidance to help you learn about and implement web accessibility for people with disabilities. Depending on your role, you might find useful tips in more than one category.
 
 {:.tips-grid .fa-ul}
 * [<span class="fa fa-li fa-paint-brush"></span><span>Designing</span> for Web Accessibility](designing.html)<span class="visuallyhidden">: </span>Tips for user interface and visual design.
-* [<span class="fa fa-li fa-pencil"></span><span>Writing</span> for Web Accessibility](writing.html)<span class="visuallyhidden">: </span>Tips for writing content.
+* [<span class="fa fa-li fa-pencil"></span><span>Writing</span> for Web Accessibility](writing.html)<span class="visuallyhidden">: </span>Tips for writing and presenting content.
```

Comment:
Could you explain why you have included 'and presenting'? I don't know that any of the writing tips talk about how to present text - but I may be confining my interpretation of 'present' to visual presentation.